app/modules/feed-items/saga/api/fetch.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import { call, put } from 'redux-saga/effects';
+
+import * as t from '../../actionTypes';
+
+import Api from '../../api';
+import { predicateTypes } from '../../model';
+import type { FeedItemType } from '../../model';
+
+// TODO: change this to topic once backend is deployed
+const mapEventTypeToPredicateType = {
+  deck_created: predicateTypes.CREATE,
+  deck_updated: predicateTypes.UPDATE,
+};
+
+const fetchSaga = function* (action: t.FetchAction): Generator<*, *, *> {
+  try {
+    const response = yield call(Api.fetch);
+
+    // eslint-disable-next-line flowtype/no-weak-types
+    const data = response.data.map((item: Object): FeedItemType => {
+      return {
+        id: item.id,
+        userId: item.relationships.user.data.id,
+        // TODO: change this to topic once backend is deployed
```

Comment:
I was testing on the current backend, which still has the old `Deck` model instead of `Topic`. When the new version is deployed, the `deck` pointer won't work anymore.